source/plot/SvgWriter2D.ooc

```diff
@@ -79,7 +79,7 @@ SvgWriter2D: class {
 					numPlotsX = 2
 				else
 					numPlotsX = 3
-				numPlotsY = Int modulo(this svgPlots count, numPlotsX) ? 1 + this svgPlots count / numPlotsX : this svgPlots count / numPlotsX
+				numPlotsY = this svgPlots count modulo(numPlotsX) ? 1 + this svgPlots count / numPlotsX : this svgPlots count / numPlotsX
```

Comment:
You didn't introduce it but we should probably not do it the C way and assume that integers have a boolean value. It is probably enough to do the lazy version for not null.